.prettierrc

```diff
@@ -1,5 +1,9 @@
 {
   "semi": false,
   "arrowParens": "always",
-  "singleQuote": true
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "endOfLine": "lf",
```

Comment:
I have end of line CRLF. If I change this in VS Code this will only work till I open some other source file with CRLF.

Reply:
Suggest  'endOfLine': 'auto',

Reply:
Suggest  'endOfLine': 'auto',

Reply:
yarn run lint --fix